---
title: "Jack Humphrey"
image: "beardy_me_2016.png"
color1: "#5A59A3"
color2: "#C66060"
angle: 130
links:
  - label: Publications
    url: "https://scholar.google.com/citations?hl=en&user=x6GuSDsAAAAJ&view_op=list_works&sortby=pubdate"
  - label: CV
    url: "https://drive.google.com/file/d/1PIUW6HT17i-G48AnCiaZ3W1_kSnI2roH/view"
  - label: Twitter
    url: "https://twitter.com/JackHumphrey_"
  - label: GitHub
    url: "https://github.com/jackhump"
  - label: Email
    url: "mailto:jack.humphrey@mssm.edu"
output:
  postcards::trestles
---

```{css, echo = FALSE}
body {
background-color: #f7eec6!important;
}
a:link {
  color: green;
  background-color: transparent;
  text-decoration: none;
}
```

I am a geneticist and computational RNA biologist working to understand neurodegeneration. I study the genetic factors that cause diseases such as amyotrophic lateral sclerosis (ALS), frontotemporal dementia (FTD), and Alzheimer's disease. I develop and apply computational tools to measure gene expression, mRNA splicing, and genetic variation, as well as statistical methods to integrate multiple types of genomic data. 

#### Education
| **University College London** | London, UK
|    _Ph.D. Clinical Neuroscience_ | September 2014 - February 2019
|       Mentors: [Adrian Isaacs](https://ukdri.ac.uk/team/adrian-isaacs), [Pietro Fratta](https://www.frattalab.org/team) & [Vincent Plagnol](https://www.linkedin.com/in/vincent-plagnol-ab8a5b65/?originalSubdomain=uk)
|    _MSci Natural Sciences_ | September 2010 - July 2014
|       Mentor: [John Wood](https://www.ucl.ac.uk/molecular-nociception-group/)
|
|
#### Experience
| **Icahn School of Medicine at Mount Sinai** | New York City, USA
|    _Instructor_ | November 2021 - Present
|    _Postdoctoral Fellow_ | February 2019 - November 2021
|       Mentor: [Towfique Raj](https://rajlab.org/)
| **Inivata** | Cambridge, UK
|    _Computational Scientist_ | January 2018 - March 2018
| **Stanford University** | Palo Alto, USA
|    _Visiting Student Researcher, [Pritchard Lab](https://web.stanford.edu/group/pritchardlab/home.html)_ | February 2017 - May 2017
| **Justus Liebig University** | Giessen, Germany
|   _DAAD RISE Scholar, [Diener Lab](https://www.uni-giessen.de/de/fbz/fb10/institute_klinikum/institute/vphysbio/allgemein/personen-veterinaer-physiologie/martin-diener)_ | June 2012 - August 2012
|
|
#### Skills
| **Programming languages I like:**
|    R, Python, Bash, AWK

| **Tools I use a lot:**
|    ggplot2, dplyr, snakemake, Shiny, STAR, samtools, vcftools, plink, tensorQTL, SQANTI, Inkscape

| **Software I wrote/co-wrote:**
|     [NYGC ALS Consortium Spinal Cord Browser](https://rstudio-connect.hpc.mssm.edu/als_spinal_cord_browser/)
|     [leafcutter](https://davidaknowles.github.io/leafcutter/)
|     [leafviz](https://github.com/jackhump/leafviz)
|     [sqtlviztools](https://github.com/jackhump/sqtlviztools) 
|     [cryptex](https://github.com/jackhump/cryptex) 
|
|
#### Selected Publications
| <u>**Humphrey J**</u>, Brophy E, Kosoy R, Zeng B, Coccia E, Mattei D, Ravi A, Efthymiou AG, Navarro E, Muller BZ, Snijders GJ, Allan A, Münch A, Kitata RB, Kleopoulos SP, Argyriou S, Shao Z, Francoeur N, Tsai CF, Gritsenko MA, Monroe ME, Paurus VL, Weitz KK, Shi T, Sebra R, Liu T, de Witte LD, Goate AM, Bennett DA, Haroutunian V, Hoffman GE, Fullard JF, Roussos P, Raj T. *Long-read RNA-seq atlas of novel microglia isoforms elucidates disease-associated genetic regulation of splicing.* (2023) [_medRxiv_](https://www.medrxiv.org/content/10.1101/2023.12.01.23299073v1)
|
| <u>**Humphrey J**</u>, Venkatesh S, Hasan R, Herb JT, de Paiva Lopes K, Küçükali F, Byrska-Bishop M, Evani US, Narzisi G, Fagegaltier D, NYGC ALS Consortium, Sleegers K, Phatnani H, Knowles DA, Fratta P, Raj T. *Integrative transcriptomic analysis of the amyotrophic lateral sclerosis spinal cord implicates glial activation and suggests new risk genes.* (2023) [_Nature Neuroscience_](https://pubmed.ncbi.nlm.nih.gov/36482247/)  
| 
| Brown A, Wilkins OG, Keuss MJ, Hill SE, Zanovello M, Lee WC, Lee FCY, Masino L, Qi YA, Bryce-Smith S, Bampton A, Gatt A, Phatnani H, NYGC ALS Consortium, Schiavo G, Fisher EMC, Raj T, Secrier M, Lashley T, Ule J, Buratti E, <u>**Humphrey J**</u>, Ward ME, Fratta P. *TDP-43 loss and ALS-risk SNPs drive mis-splicing and depletion of UNC13A.* (2022) [_Nature_](https://pubmed.ncbi.nlm.nih.gov/35197628/) 
|
| Hasan R<sup>†</sup>, <u>**Humphrey J**</u><sup>†</sup>, Bettencount C, NYGC ALS Consortium, Lashley T, Fratta P, Raj T. *Transcriptomic analysis of frontotemporal lobar degeneration with TDP-43 pathology reveals cellular alterations across multiple brain regions.* (2022) [_Acta Neuropathologica_](https://pubmed.ncbi.nlm.nih.gov/34961893/)  
| 
| Lopes KP<sup>†</sup>, Snijders GJL<sup>†</sup>, <u>**Humphrey J**</u><sup>†</sup>, Allan A, Sneeboer M, Navarro E, Schilder BM, Vialle RA, Parks M, Missall R, van Zuiden W, Gigase F, Kübler R, van Berlekom AB, Böttcher C, Priller J, Kahn RS, de Witte LD, Raj T. *Genetic analysis of the human microglial transcriptome across brain regions, aging and disease pathologies.* (2022) [_Nature Genetics_](https://pubmed.ncbi.nlm.nih.gov/34992268/)

